code/datums/emergency_calls/emergency_call.dm

```diff
@@ -83,7 +83,17 @@
 	if(SSticker?.mode?.waiting_for_candidates) //It's already been activated
 		return FALSE
 
-	picked_call.mob_max = rand(5, 15)
+	if(istype(src, /datum/game_mode/colonialmarines) ) //If we're fighting benos
```

Comment:
This change won't be compatible with 2 other prs: #1072 and #1130 plus this would better serve as a redefinition than to snowflake it like this.

Reply:
FFS, I give up.

Reply:
Just make it a redefinition and I'll handle the conflicts on my end